charts/emissary-ingress/templates/rbac.yaml

```diff
@@ -110,6 +115,10 @@ rules:
     resources: [ "ingresses/status" ]
     verbs: ["update"]
 
+  - apiGroups: [ "apiextensions.k8s.io" ]
+    resources: [ "*" ]
+    verbs: ["get", "list", "watch", "update", "patch", "create", "delete" ]
+    
```

Comment:
1. Does this really need to be `resources: [ "*" ]` instead of just `resources: [ "CustomResourceDefinitions" ]`?
2. Does this really need all of those verbs, and not just get/list/update?